roles/docker/tasks/systemd.yml

```diff
@@ -12,9 +12,9 @@
   when: http_proxy is defined or https_proxy is defined
 
 - name: get systemd version
-  command: rpm -q --qf '%{V}\n' systemd
+  command: systemctl --version | head -n 1 | cut -d " " -f 2
   register: systemd_version
-  when: ansible_os_family == "RedHat" and not is_atomic
+  when: not is_atomic
```

Comment:
won't this cause issues for atomic in the service template below since `systemd_version` will be undefined?

Reply:
True. It seems I can put a check for is_atomic in the docker service check. I'll update this PR tomorrow. Thanks.

Reply:
@bradbeam I've updated the template to check for atomic. Thanks for catching that.